rosbag2_cpp/src/rosbag2_cpp/message_definitions/local_message_definition_source.cpp

```diff
@@ -187,8 +195,18 @@ const LocalMessageDefinitionSource::MessageSpec & LocalMessageDefinitionSource::
     ROSBAG2_CPP_LOG_WARN("'%s'", e.what());
     throw DefinitionNotFoundError(definition_identifier.topic_type());
   }
-  std::string dir = definition_identifier.format() == Format::MSG ||
-    definition_identifier.format() == Format::IDL ? "/msg/" : "/srv/";
+  std::string dir;
+  if (definition_identifier.format() == Format::MSG ||
+    definition_identifier.format() == Format::IDL)
+  {
+    dir = "/msg/";
+  } else if (definition_identifier.format() == Format::SRV) {
+    dir = "/srv/";
+  } else if (definition_identifier.format() == Format::ACT) {
+    dir = "/action/";
+  } else {
+    throw std::runtime_error("Unknown format type");
```

Comment:
Minor: could we include the format type in the error message?

Reply:
Here, unknown format type should be `Format::UNKNOWN`.